### PR TITLE
Mayhem - GH actions integrated

### PR DESCRIPTION
**Pull request by kiranprasad**

### CODE REVIEW

File: fuzz/fuzz_targets/chacha20.rs [hunk 1]
```diff
@@ -27,7 +27,8 @@ fuzz_target!(|data: ([u8; KEY_LEN], [u8; NONCE_LEN])| {
 		let mut keystream: Vec<u8> = Vec::with_capacity(reference_keystream.len());
 
 		while reference_keystream.len() > keystream.len() {
-			chacha::chacha_block(20, state)
```
Comment (xansec) [inline fuzz/fuzz_targets/chacha20.rs]: Good eye!

**Review by xansec — CHANGES_REQUESTED** [review]
I don't think the upstream will want to have the `nanorand-fuzz`->`fuzz` directory name change; I would revert the name change unless there is some specific reason this change was made?

**Comment by kiranprasad** [comment]
I was facing issues with the fuzzing directory not getting picked up by the cargo-fuzz build with a custom name before. 
Now resolved with the updated version.

**Comment by xansec** [comment]
LGTM!

**Pull request merged**